fixtures/keywords/kotlin/src/lib.rs

```diff
@@ -56,4 +56,13 @@ pub enum fun {
     class { object: u8 },
 }
 
+// `FooError` is turned into `FooException`, so this enum will end up
+// being named just `Exception`. Without special care, this will clash
+// with `kotlin.Exception` class.
+#[derive(Debug, thiserror::Error)]
```

Comment:
I think this is just missing the uniffi derive?

Reply:
Oh, thanks! I've now tried [with the derive](https://app.circleci.com/pipelines/github/mozilla/uniffi-rs/4289/workflows/7c0d86e1-8069-44f8-a428-45799ae727f4) and with [changes to the UDL file](https://app.circleci.com/pipelines/github/mozilla/uniffi-rs/4290/workflows/339d1fd6-814e-41fe-8c40-59ef4315a365) and both work.

If I understand correct, it should be okay to mix proc-macros and the UDL file, so I would have expected both of them to fail?